easybuild/easyconfigs/q/Qt6/Qt6-6.5.2_fix_OF-Gentoo.patch

```diff
@@ -0,0 +1,48 @@
+############################################
+# Update from Qt5-5.14.1_fix-OF-Gentoo.patch
+# 1. qt3d/..../unzip/ioapi.h and qtquick3d/..../unzip/ioapi.h do not use OF macro anymore
+# 2. qtquick3d/..../minizip/ioapi.h is moved to qt3d/..../minizip/ioapi.h
+############################################
+fix compilation on top of zlib provided by Gentoo,
+where OF macro has been renamed to _Z_OF
+see https://bugreports.qt.io/browse/QTBUG-68467 and https://bugs.gentoo.org/383179
+author: Kenneth Hoste (HPC-UGent)
+diff --git a/qt3d/src/3rdparty/assimp/src/contrib/zlib/contrib/minizip/ioapi.h b/qt3d/src/3rdparty/assimp/src/contrib/zlib/contrib/minizip/ioapi.h
+index 8dcbdb06e3..e9cc96914e 100644
+--- a/qt3d/src/3rdparty/assimp/src/contrib/zlib/contrib/minizip/ioapi.h
++++ b/qt3d/src/3rdparty/assimp/src/contrib/zlib/contrib/minizip/ioapi.h
+@@ -130,6 +130,14 @@ extern "C" {
+ #endif
+
+
++// if OF macro is not defined by _Z_OF is, define OF as _Z_OF
```

Comment:
Should probably be "if OF macro is not defined but _Z_OF is"

And the same for the one below.

Reply:
Yeah it should be. I copied this paragraph from the older patch but did not check

Reply:
Can you please make a commit for these "by" -> "but" soon. I'd like to get this one merged